Example/Tests/GINICameraViewControllerTests.swift

```diff
@@ -10,10 +10,15 @@ final class CameraViewControllerTests: XCTestCase {
         let imageData = UIImageJPEGRepresentation(image!, 0.9)!
         return imageData
     }()
+    var giniConfiguration: GiniConfiguration = GiniConfiguration.sharedConfiguration
+    lazy var screenAPICoordinator = GiniScreenAPICoordinator(withDelegate: nil,
+                                                             giniConfiguration: self.giniConfiguration)
     
     override func setUp() {
         super.setUp()
-        vc = CameraViewController(successBlock: { _ in }, failureBlock: { _ in })
+        giniConfiguration.multipageEnabled = true
+        vc = CameraViewController(giniConfiguration: giniConfiguration)
```

Comment:
Is there a reason why the `GiniConfiguration` and `screenAPICoordinator` are initialized when they are declared, but the `CameraViewController` is created in he setup method?

Just a question of consistency.

Reply:
The `screenAPICoordinator` is initialized when it was declared because I don't need another instance of it, making the tests slightly faster, although I guess that It is no saving a lot of time. I can move it to the initialization method.